gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
+var replace = require('gulp-replace');
+var pjson = require('./package.json');
+
+gulp.task('propagate-version', function() {
+  gulp.src(['./bower.json'])
+    .pipe(replace(/"version": "\d+\.\d+\.\d+"/, '"version": "' + pjson.version + '"'))
+    .pipe(gulp.dest('./'));
+
+  gulp.src(['./juicy-filedrop.html'])
```

Comment:
I hate such complex regexp's. I have a simpler idea, I will use `bower.json` as the source of the previous version number!